Dockerfile

```diff
@@ -14,4 +14,5 @@ RUN \
 
 EXPOSE 9000
 ENTRYPOINT ["go-wrapper", "run"]
+CMD ["server"]
```

Comment:
Distributed release brings in new features like 'control, healing' if we default this it complicates the documentation. It was a mistake to keep "server" as part of the entry point previously so we had to fix the bug which led to a breaking change. I apologize for that :( but as it stands today is the correct style of the docker image we wanted. It behaves exactly like our released binaries.